crates/hyperswitch_connectors/src/connectors/nexixpay/transformers.rs

```diff
@@ -371,53 +435,119 @@ impl TryFrom<&NexixpayRouterData<&PaymentsAuthorizeRouterData>> for NexixpayPaym
     fn try_from(
         item: &NexixpayRouterData<&PaymentsAuthorizeRouterData>,
     ) -> Result<Self, Self::Error> {
-        match item.router_data.request.payment_method_data {
-            PaymentMethodData::Card(ref req_card) => {
-                let card = NexixpayCard {
-                    pan: req_card.card_number.clone(),
-                    expiry_date: req_card.get_expiry_date_as_mmyy()?,
-                };
-                let billing_address = Address {
-                    name: item.router_data.get_billing_full_name()?,
-                    street: item.router_data.get_billing_line1()?,
-                    city: item.router_data.get_billing_city()?,
-                    post_code: item.router_data.get_billing_zip()?,
-                    country: item.router_data.get_billing_country()?,
-                };
-                let customer_info = CustomerInfo {
-                    card_holder_name: item.router_data.get_billing_full_name()?,
-                    billing_address: billing_address.clone(),
-                    shipping_address: Some(billing_address),
-                };
-                let order = Order {
-                    order_id: item.router_data.connector_request_reference_id.clone(),
-                    amount: item.amount.clone(),
-                    currency: item.router_data.request.currency,
-                    description: item.router_data.description.clone(),
-                    customer_info,
+        let billing_address = Address {
+            name: item.router_data.get_billing_full_name()?,
+            street: item.router_data.get_billing_line1()?,
+            city: item.router_data.get_billing_city()?,
+            post_code: item.router_data.get_billing_zip()?,
+            country: item.router_data.get_billing_country()?,
+        };
```

Comment:
All required fields should be added in payment_connector_required_fields.rs file